bot/exts/moderation/voice_gate.py

```diff
@@ -144,8 +155,13 @@ async def voice_verify(self, ctx: Context, *_) -> None:
                     color=Colour.red()
                 )
                 log.warning(f"Got response code {e.status} while trying to get {ctx.author.id} Metricity data.")
-
-            await ctx.author.send(embed=embed)
+            try:
+                await ctx.author.send(embed=embed)
+            except discord.Forbidden:
+                log.info("Could not send user DM. Sending in voice-verify channel and scheduling delete.")
+                message = await ctx.send(embed=embed)
+                await asyncio.sleep(GateConf.voice_ping_delete_delay)
+                await message.delete()
```

Comment:
Ah, it seems we don't need to do this since it already gets deleted here https://github.com/python-discord/bot/blob/67934bd582ae5a6fac459c02ba3fa6c17d085d1a/bot/exts/moderation/voice_gate.py#L207-L214